ionoscloud.go

```diff
@@ -865,33 +839,74 @@ func (d *Driver) Create() (err error) {
 		ipsForAttachedNic = reservedIps // For public NICs we use the generated IPs
 	}
 
+	attachedNics := sdkgo.NewNicsWithDefaults()
+
 	lanId, _ := strconv.Atoi(d.LanId)
-	nic, err := d.client().CreateAttachNIC(d.DatacenterId, d.ServerId, d.MachineName, d.NicDhcp, int32(lanId), ipsForAttachedNic)
+	lanId_int32 := int32(lanId)
+	nicProperties := &sdkgo.NicProperties{
+		Name: &d.MachineName,
+		Lan:  &lanId_int32,
+		Ips:  ipsForAttachedNic,
+		Dhcp: &d.NicDhcp,
+	}
+
+	attachedNics.Items = &[]sdkgo.Nic{
+		{
+			Properties: nicProperties,
+		},
+	}
+
+	for _, additionalLanId := range d.AdditionalLansIds {
+		dhcp_true := true
+		additionalLanId_int32 := int32(additionalLanId)
+		additionalNicName := d.MachineName + " " + fmt.Sprint(additionalLanId)
+		nicProperties = &sdkgo.NicProperties{
+			Name: &additionalNicName,
+			Lan:  &additionalLanId_int32,
+			Ips:  nil,
+			Dhcp: &dhcp_true,
```

Comment:
just a nitpick but we have SDK util funcs that help us get rid of these variables:

I mean we can do `Dhcp: sdkgo.ToPtr(true)` instead of `Dhcp: &dhcp_true`